barretenberg/cpp/src/barretenberg/protogalaxy/protogalaxy_prover_internal.hpp

```diff
@@ -25,7 +25,12 @@ template <class ProverInstances_> class ProtogalaxyProverInternal {
     using ProverPolynomials = typename Flavor::ProverPolynomials;
     using Relations = typename Flavor::Relations;
     using RelationSeparator = typename Flavor::RelationSeparator;
-    using CombinedRelationSeparator =
+    static constexpr size_t NUM_INSTANCES = ProverInstances_::NUM;
```

Comment:
This is a good idea but I think we need this because elsewhere we refer to the type `ProtogalaxyProverInternal::ProverInstances`. In any case this class is likely to go away.